README.md

```diff
@@ -488,15 +488,51 @@ Avoid including a key without a meaningful value:
 
 # Timeouts
 
-## TODO
+Generally APIs should respond in less than 250ms on a wired connection. There needs to be a special reason for exeeding that. And the developer needs to understand that it will be harder / more expensive to scale the backend if response times are high.
+
+It's common that the webserver configuration will timeout the request in 30 or 60 sec. 
+
+## Client to server
+
+Since you never know what network the client is on, if they are in a metro or 100mBit wifi. Response time can very a lot for several reason.
```

Comment:
```suggestion
Since you never know what network the client is on, if they are in a metro or 100mBit wifi. Response time can vary a lot for several reason.
```